core/src/main/resources/static/management/user/buttonsAJAX.js

```diff
@@ -106,14 +106,58 @@ function otherCheck() {
     }
 }
 
+// Save button in editUserModal
+function updateUser(userId) {
+    clearAllErrorsSpan();
+    let formData = $('#editUserForm' + userId).serializeArray().reduce(function (obj, item) {
+        obj[item.name] = item.value;
+        return obj;
+    }, {});
+    validateForUpdate(formData, userId);
+    let returnData = {
```

Comment:
It's the payload/data, not the return data from server. Consider changing the name

Reply:
Oh yeah, I know. I must have messed that up while refactoring